requirements.in

```diff
@@ -12,3 +12,4 @@ requests_kerberos
 requests
 semver
 setuptools
+prometheus-flask-exporter
```

Comment:
shouldn't be this added to `requirements-web.in` instead?

Reply:
Moved.